### PR TITLE
Work and about myself

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
Finish setup with works section and about myself section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,17 +24,127 @@ <h1 class="main_content">
...
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/ligth.svg"> </div>
+                            <h3 class="text3-about">Skills</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                </div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap all these two containers in one section as all hold the information about one section which is `About Myself`. It's better and required all of them in one `section` conatiner.

File: index.html [hunk 2]
```diff
@@ -24,17 +24,127 @@ <h1 class="main_content">
...
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/Icons.svg"></div>
+                            <h3 class="text3-about">Frameworks</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
```
Comment (levy002) [inline index.html:135]: - [ ] Kindly make sure to add the `border-radius` on each technology background. in order to look like the provided Figma design. :wink: 

  - ### YOUR DESIGN
  
    image

  - ### FIGMA DESIGN
  
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Alejandro-Bernal-M   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]

**Comment by Alejandro-Bernal-M** [comment]
Hello @levy002 , thank you for the review, I corrected the section "section-about" to embrace all the "about" items and made changes at the border-radius to be equal to Figma's template, I also made corrections to responsiveness

File: index.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
```
Comment (Kamba56) [inline index.html:8]: - [Optional] Consider using an indentation of two spaces in all your documents, this is a best practice and also a convention.

**Review by Kamba56 — APPROVED** [review]
Hi @Alejandro-Bernal-M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**